### PR TITLE
Adapt the internationalization page

### PR DESCRIPTION
**Pull request by scarletfog**
### Context
Adapt the internationalization page accordingly to the changes brought by modularization.

### Types of changes
- [x] Improvement (better description, more examples etc.)

### Related issue(s):
1. #270

### CODE REVIEW

**Comment by scarletfog** [comment]
@wojciechczerniak could you please have a look?

I am wondering about "any locale code", for example: 
`import xxXX from 'handsontable/i18n/languages/xx-XX';`

it was in the file Piotrek provided me with but now I have some second thoughts. It stands out, and more importantly - we don't have locales like that(?), all of them are more like `XY-xy`. Still looks strange a bit. Maybe we should stick to one locale as an example?

**Comment by scarletfog** [comment]
Ok, as discussed during the call - we will keep an example locale code.

File: tutorials/internationalization.html [hunk 1]
```diff
@@ -31,32 +30,80 @@ <h3 id="overview">Overview</h3>
...
+
+const hot = new Handsontable(container, {
+  language: daDK.languageCode,
+});
+        </code></pre>
+      </li>
+
+      <li>
+        <h4>Universal Module Definition (UMD)</h4>
+        <p>Languages included this way are ready to use immediately after loading the file.</p>
```
Comment (wojciechczerniak) [inline tutorials/internationalization.html:73]: We should mention why, they are auto-registered.

Comment (scarletfog) [reply tutorials/internationalization.html:73]: @swistach help!

File: tutorials/internationalization.html [hunk 2]
```diff
@@ -354,62 +316,97 @@ <h4>Language file</h4>
...
-      which are described briefly in the next section.
+      <code>registerLanguageDictionary</code> method and the static constant <code>dictionaryKeys</code>
+      which are described briefly in the next section. ()
     </p>
 
-    <pre><code class="lang-js">const C = Handsontable.languages.dictionaryKeys;
+    <pre><code class="lang-js">
+import Handsontable from 'handsontable/base';
+import { registerLanguageDictionary, dictionaryKeys } from 'handsontable/i18n';
+import daDK from 'handsontable/i18n/languages/daDK';
```
Comment (budnix) [inline tutorials/internationalization.html:330]:
```suggestion
import { 
  registerLanguageDictionary, 
  dictionaryKeys,
  daDK
} from 'handsontable/i18n';
```

File: tutorials/internationalization.html [hunk 3]
```diff
@@ -354,62 +316,97 @@ <h4>Language file</h4>
...
-});</code></pre>
+    <h4>Overwrite language in app</h4>
+
+    <p>
+      By default, built-in translations use <code>en-US</code> as a fallback option if in your translation some keys are missing.
+    </p>
+    <pre><code class="lang-js">
+import Handsontable from 'handsontable/base';
+import { registerLanguageDictionary, dictionaryKeys } from 'handsontable/i18n';
+import daDK from 'handsontable/i18n/languages/daDK';
```
Comment (budnix) [inline tutorials/internationalization.html:347]:
```suggestion
import { 
  registerLanguageDictionary, 
  dictionaryKeys,
  daDK
} from 'handsontable/i18n';
```

File: tutorials/internationalization.html [hunk 4]
```diff
@@ -81,20 +107,26 @@ <h3 id="demo">Demo</h3>
...
+        const dataObj = [
           ['Lorem', 'ipsum', 'dolor', 'sit', '12/1/2015', 23],
           ['adipiscing', 'elit', 'Ut', 'imperdiet', '5/12/2015', 6],
           ['Pellentesque', 'vulputate', 'leo', 'semper', '10/23/2015', 26],
           ['diam', 'et', 'malesuada', 'libero', '12/1/2014', 98],
           ['orci', 'et', 'dignissim', 'hendrerit', '12/1/2016', 8.5]
         ];</script>
       <script data-jsfiddle="example1">
-        var example1 = document.getElementById('example1');
+        import Handsontable from 'handsontable/base';
```
[the remark this answers is not in the document]
Comment (scarletfog) [reply tutorials/internationalization.html:116]: > We can show the new syntax by placing the code into the `<code>` tags

So should we get rid of the "interactive" example and use a simple "copy and paste" one?

[the remark this answers is not in the document]
Comment (wojciechczerniak) [reply tutorials/internationalization.html:116]: A working demo should stay. We have other examples that use the correct syntax.

File: tutorials/internationalization.html (hunk 1 above)
Comment (swistach) [reply tutorials/internationalization.html:71]: so maybe:
```
<p>Languages included this way are ready to use immediately after loading the file. Each file contains UMD loader which looks for <code>Handsontable</code> in a global/externals context. If <code>Handsontable</code> is available then register itself to the proper context.</p>
```

**Review by wojciechczerniak — APPROVED** [review]
👍

**Pull request merged**